ggml.c

```diff
@@ -10817,6 +10818,27 @@ static void ggml_compute_forward_mul_mat(
     }
 #endif
 
+    if (src1_cont) {
+        for (int64_t j = 0; j < ne13; j++)
+            for (int64_t i = 0; i < ne12; i++)
+                if (!llamafile_sgemm(ne01, ne11, ne00/ggml_blck_size(src0->type),
+                                     (const char *)src0->data + i/r2*nb02 + j/r3*nb03,
+                                     nb01/ggml_type_size(src0->type),
+                                     (const char *)src1->data + i*nb12 + j*nb13,
+                                     nb11/ggml_type_size(src1->type),
+                                     (char *)dst->data + i*nb2 + j*nb3,
+                                     nb1/ggml_type_size(dst->type),
+                                     ith, nth,
+                                     params->type,
+                                     src0->type,
+                                     src1->type,
+                                     dst->type))
+                    goto UseGgmlGemm1;
+        return;
+    }
+UseGgmlGemm1:
+    (void)0;
```

Comment:
`;` ?

Reply:
That's to avoid the compiler complaining if the label comes before a variable declaration.

Reply:
Yes. But using just `;` is a simple way to achieve the same goal without introducing dummy expressions. Just do:

```
UseGgmlGemm1: ;
```

BTW. This issue with labels is fixed in C23 standard.

Reply:
Thank you @tstanisl, you taught me something I didn't know. Fixed!